src/Domain/ValetToken/ValetTokenGenerator.ts

```diff
@@ -0,0 +1,24 @@
+import { CrypterInterface as Crypter } from '../Encryption/CrypterInterface'
+import { ValetPayload, ValetToken } from './ValetToken'
+
+export type JwtSecret = string
+export type ValetTokenSecret = string
+
+/**
+ * Generates a valet key for given user `uuid` that allows them to perform the `permittedOperation` on the `permittedResources` in the data store.
+ */
+export type GenerateValetToken = ({
```

Comment:
apply previous comments

Reply:
I assume this is the same point as in: https://github.com/standardnotes/files/pull/3#discussion_r598958158.

Will remove the comment.